example/googleapis/pubspec.yaml

```diff
@@ -3,13 +3,13 @@ description: Dart gRPC client sample for Google APIs
 homepage: https://github.com/dart-lang/grpc-dart
 
 environment:
-  sdk: '>=1.24.3 <2.0.0'
+  sdk: '>=2.0.0-dev.58.0 <3.0.0'
```

Comment:
With Dart 2.0 now released, could we now set all the SDK requirements to >=2.0.0 <3.0.0?